rts/lra-examples/cdi-participant-with-coordinator/pom.xml

```diff
@@ -33,30 +34,48 @@
                 <enabled>true</enabled>
             </snapshots>
         </repository>
+<!--
+        <repository>
+            <id>jboss-public-releases-repository-group</id>
+            <name>JBoss Public Releases Maven Repository Group</name>
+            <url>https://repository.jboss.org/nexus/content/repositories/releases</url>
+        </repository>
+-->
```

Comment:
this may confuse users who are studying quickstart code in detail. Can you remove it please?

Reply:
I removed it